scripts/upload-assets.py

```diff
@@ -28,12 +28,17 @@
 dir_url = ("https://storage.googleapis.com/tilt-static-assets/%s/" % version)
 url = dir_url + "index.html"
 print("Uploading to %s" % dir_url)
-status = subprocess.call([
-  "gsutil", "stat", "gs://tilt-static-assets/%s/index.html" % version
-])
-if status == 0:
-  print("Error: Files already exist: %s" % url)
-  sys.exit(1)
+
+if not args.no_check:
```

Comment:
is there a risk this will get us into a weird state where two releases are on top of each other? do we need to delete the old folder if it exists?